commands/project_init.go

```diff
@@ -0,0 +1,76 @@
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/fatih/color"
+	"github.com/platformsh/platformify/commands"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	ProjectInitCmd.SetHelpFunc(projectInitHelpFn)
+	RootCmd.AddCommand(ProjectInitCmd)
+}
+
+var ProjectInitCmd = &cobra.Command{
+	Use:           "project:init",
+	Short:         "Initialize the needed YAML files for your Platform.sh project",
+	Aliases:       []string{"ify"},
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	RunE:          commands.PlatformifyCmd.RunE,
+}
+
+func projectInitHelpFn(cmd *cobra.Command, _ []string) {
+	var b bytes.Buffer
+	writer := tabwriter.NewWriter(&b, 0, 8, 1, ' ', 0)
+
+	c := ProjectInitCommand
+	fmt.Fprintln(writer, color.YellowString("Command: ")+c.Name.String())
+	fmt.Fprintln(writer, color.YellowString("Description: ")+c.Description.String())
+	fmt.Fprintln(writer, "")
+	if len(c.Usage) > 0 {
+		fmt.Fprintln(writer, color.YellowString("Usage:"))
+		for _, usage := range c.Usage {
+			fmt.Fprintln(writer, " "+usage)
+		}
+		fmt.Fprintln(writer, "")
+	}
+	if c.Definition.Arguments != nil && c.Definition.Arguments.Len() > 0 {
+		fmt.Fprintln(writer, color.YellowString("Arguments:"))
+		for pair := c.Definition.Arguments.Oldest(); pair != nil; pair = pair.Next() {
+			arg := pair.Value
+			fmt.Fprintf(writer, "  %s\t%s\n", color.GreenString(arg.Name), arg.Description)
+		}
+		fmt.Fprintln(writer, "")
+	}
+	if c.Definition.Options != nil && c.Definition.Options.Len() > 0 {
+		fmt.Fprintln(writer, color.YellowString("Options:"))
+		for pair := c.Definition.Options.Oldest(); pair != nil; pair = pair.Next() {
+			opt := pair.Value
+			shortcut := opt.Shortcut
+			if shortcut == "" {
+				shortcut = "   "
+			} else {
+				shortcut += ","
+			}
+			fmt.Fprintf(writer, "  %s %s\t%s\n", color.GreenString(shortcut), color.GreenString(opt.Name), opt.Description)
+		}
+		fmt.Fprintln(writer, "")
+	}
+	if len(c.Examples) > 0 {
+		fmt.Fprintln(writer, color.YellowString("Examples:"))
```

Comment:
Our Legacy CLI has a "Help" section instead of examples, you should use the same format here as well.

This is from `platform help project:create`

```
Help:
 Use this command to create a new project.
 
 An interactive form will be presented with the available options. But if the
 command is run non-interactively (with --yes), the form will not be displayed,
 and the --region option will be required.
 
 A project subscription will be requested, and then checked periodically (every 3
 seconds) until the project has been activated, or until the process times out
 (after 15 minutes by default).
 
 If known, the project ID will be output to STDOUT. All other output will be sent
 to STDERR.
```

Reply:
actually, Legacy-CLI has both sections - help and examples.
for example, `platform help environment:delete`
I added the Help section into common function for generating help pages, but left examples of `project:init` in the Examples section.
if you still want me to move it from Examples to Help, let me know, I will transfer it

Reply:
Nope, that's cool. I hadn't seen that both sections exist sorry 😇